### ISSUE TITLE
CEngineFuncs::Voice_GetClientListening has wrong return type

### ISSUE THREAD

**Issue by fnky** [baso88/SC_AngelScript#33]
`CEngineFuncs::Voice_GetClientListening` has the wrong return type. It should be `bool`, not `void`.

**Comment by fnky** [comment]
Code to fix this:

```cpp
pDocumentation->RegisterObjectMethod( "Voice get client listening", 
    pszObjectName, "bool Voice_GetClientListening(int iReceiver, int iSender)", 
    asMETHOD( CASEngineFuncs, Voice_GetClientListening ), asCALL_THISCALL );
```

In `trunk/dlls/Angelscript/ScriptTypes/ASEngineFuncs.cpp`

**Comment by baso88** [comment]
The fix for this issue will be included in the next update (5.13). Closing.
